src/openzeppelin/tests/access/test_accesscontrol.cairo

```diff
@@ -320,10 +353,18 @@ fn test_renounceRole_for_role_not_granted() {
 fn test_renounce_role_for_granted_role() {
     let mut state = setup();
     testing::set_caller_address(ADMIN());
+
     AccessControlImpl::grant_role(ref state, ROLE, AUTHORIZED());
+    testing::pop_log_raw(ZERO());
 
     testing::set_caller_address(AUTHORIZED());
     AccessControlImpl::renounce_role(ref state, ROLE, AUTHORIZED());
+
+    let event = testing::pop_log::<RoleRevoked>(ZERO()).unwrap();
+    assert(event.role == ROLE, 'Invalid role');
+    assert(event.account == AUTHORIZED(), 'Invalid account');
+    assert(event.sender == AUTHORIZED(), 'Invalid sender');
```

Comment:
shall we abstract this and check for extra events/extra parameters as in [the Ownable PR](https://github.com/OpenZeppelin/cairo-contracts/pull/675)?

Reply:
i just noticed we're not following the same order: sometimes we test camel first, then snake, and sometimes the other way around